apps/new-tab/src/Widgets.tsx

```
@@ -235,672 +104,96 @@ function TodoWidget() {
+                  draggable="true"
+                  onDragStart={(e) => handleDragStart(e, task.id)}
+                  onDragOver={handleDragOver}
+                  onDragEnd={handleDragEnd}
+                  onDrop={(e) => handleDrop(e, task.id)}
+                  class="task hover:bg-accent/20 flex cursor-move items-center gap-2 rounded p-1
+                    transition-colors"
+                  ondblclick={(e) => {
+                    (e.target as HTMLLabelElement).contentEditable = "true";
+                    (e.target as HTMLLabelElement).focus();
                   }}
                 >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)